src/components/ValueCard/ValueCard.test.jsx

```diff
@@ -37,4 +40,31 @@ describe('ValueCard', () => {
         .prop('layout')
     ).toEqual(CARD_LAYOUTS.VERTICAL);
   });
+
+  test('DataState prop shows DataState elements instead of content', () => {
+    const wrapperWithoutDataState = mount(
+      <ValueCard
+        title="Health score"
+        content={{ attributes: [{ label: 'title', dataSourceId: 'v' }] }}
+        size={CARD_SIZES.SMALL}
+        values={{ v: 'value' }}
+      />
+    );
+    expect(wrapperWithoutDataState.find(`.${iotPrefix}--data-state-container`)).toHaveLength(0);
+
+    const wrapperWithDataState = mount(
+      <ValueCard
+        title="Health score"
+        content={{ attributes: [{ label: 'title', dataSourceId: 'v' }] }}
+        size={CARD_SIZES.SMALL}
+        dataState={{
```

Comment:
does this work for multi-value cards?

Reply:
Yes, if you add the dataState property it will be activated regardless of how many values you added. It overrides all other rendering.

Reply:
Yeah, but the dataState property is set as a single object for the entire Card.  So what happens if I can load 2 out 3 pieces of data?  How each each value rendering get the error state, since, 2 will be good and show a value and the third might have an error?  Or in the case of a value card is it's an error state for all values even if 1 fails.  ie, it's the card that has an error state and shows and error vs each value in a multi-value card.

Reply:
It is implemented as as error-state (or empty-state) for the ValueCard itself, not for specific values. It will be up to the app to decide how to handle the case of 1/3 items error, if you want to show the 2 items that didn't error you cannot use the dataState in its current implementation.  Error state per value would become much more complex, it is doable, but not how I interpreted the spec. @davidicus What is your take on this?